src/stability_sdk/animation_ui.py

```diff
@@ -0,0 +1,785 @@
+import glob
+import json
+import locale
+import os
+import param
+import shutil
+import traceback
+
+from collections import OrderedDict
+from PIL import Image
+from tqdm import tqdm
+from typing import Any, Dict, List, Optional
+
+try:
+    import gradio as gr
+except ImportError:
+    raise ImportError(
+        "Failed to import animation UI requirements. To use the animation UI, install the dependencies with:\n" 
+        "   pip install --upgrade stability_sdk[anim_ui]"
+    )
+
+from .api import (
+    ClassifierException, 
+    Context,
+    OutOfCreditsException,
+)
+from .animation import (
+    AnimationArgs,
+    Animator,
+    AnimationSettings,
+    BasicSettings,
+    CameraSettings,
+    CoherenceSettings,
+    ColorSettings,
+    DepthSettings,
+    InpaintingSettings,
+    Rendering3dSettings,
+    VideoInputSettings,
+    VideoOutputSettings,
+    create_video_from_frames,
+    interpolate_frames
+)
+from .utils import interpolate_mode_from_string
+
+
+DATA_VERSION = "0.1"
+DATA_GENERATOR = "alpha-test-notebook"
```

Comment:
we should change "alpha-test-notebook" to a more appropriate value prior to merge

Reply:
Definitely, good catch!

Reply:
proposed new value: `stability_sdk.animiation_ui` 
thoughts?

Reply:
It's a creative spelling of animation, how about `stability_sdk.animation_ui` instead? 😄